test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -24,6 +24,7 @@ namespace Microsoft.ML.Runtime.RunTests
     /// </summary>
     public abstract partial class BaseTestBaseline : BaseTestClass, IDisposable
     {
+        public const decimal tolerance = 10000000;
```

Comment:
Naming convention is UpperCamel for `public const`.  

Examples: https://github.com/dotnet/machinelearning/search?q=%22public+const%22&unscoped_q=%22public+const%22